### DISCUSSION TITLE
Missed translations

### DISCUSSION THREAD

**Discussion by KOLIWAN** [fast4x/RiMusic#2305] [General]
Hi guys,
I suggest creating a discussion with missed translations. It's not convenient to create a separate bug report for each line found, so I think it's a good idea to leave them here.

**Comment by KOLIWAN** [comment]
I'll start. It's missing string "Cache location" in Crowdin, please add it :)

**Reply by KOLIWAN** [reply]
Also missed string "Private cache location, can't be cleaned by the system"

**Reply by fast4x** [reply]
Sorry, happens to forget something 😂😂

**Reply by fast4x** [reply]
Thanks

**Reply by KOLIWAN** [reply]
Also found missed string "BACKUP AND RESTORE" in Data settings

**Reply by KOLIWAN** [reply]
Hey Fast, "Cache location" and "Private cache location, can't be cleaned by the system" are still missed :( Can you fix it please?

**Comment by fast4x** [comment]
Done

**Comment by KOLIWAN** [comment]
Hey @fast4x, there's missing translation if "Show Stats for Nerds in Player" is enabled
There's no option to translate "Quality" in Crowdin

**Reply by fast4x** [reply]
I haven't found untranslated strings....

**Reply by KOLIWAN** [reply]
![Screenshot_20240728_133453.jpg](https://github.com/user-attachments/assets/20320268-76a2-4c53-9b6b-6af2b5049694)

This one

**Reply by KOLIWAN** [reply]
Fast can you please reopen the discussion.. Im going to share missed translations in the future

**Reply by aneesh1122** [reply]
The old 'show stats for nerds' did not have string for 'Quality' so I did not add it here either. I'll add it.

**Comment by fast4x** [comment]
Reopened

**Comment by KOLIWAN** [comment]
Hey Fast,  I found out that there's no translations for "TRANSLATORS" and "DEVELOPERS/DESIGNERS"

![Screenshot_20241020_095825.jpg](https://github.com/user-attachments/assets/419e57a4-c25f-471d-ab7c-b9d1e87493ac)

![Screenshot_20241020_095819.jpg](https://github.com/user-attachments/assets/3df785b0-64b9-4b58-8a1e-d09023914f91)

**Reply by fast4x** [reply]
Hi, ok I'll add soon thanks

**Reply by fergassss1** [reply]
> Hey Fast, I found out that there's no translations for "TRANSLATORS" and "DEVELOPERS/DESIGNERS"
> 
> ![Screenshot_20241020_095825.jpg](https://private-user-images.githubusercontent.com/76595855/378121772-419e57a4-c25f-471d-ab7c-b9d1e87493ac.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4ODU0NTAsIm5iZiI6MTcyOTg4NTE1MCwicGF0aCI6Ii83NjU5NTg1NS8zNzgxMjE3NzItNDE5ZTU3YTQtYzI1Zi00NzFkLWFiN2MtYjlkMWU4NzQ5M2FjLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI1VDE5MzkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyMDYwYzlhOTU1ZDQ4Y2Q2MDY2MjRmOGI0NzFhODlkYzg3Njc4NmQxZWVjODllMDAwYzFjMTg0YThjNWFkMzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VCpdq3awUSYyXndry5O7mOVOZj063i-0wUSxVXmHmj8)
> 
> ![Screenshot_20241020_095819.jpg](https://private-user-images.githubusercontent.com/76595855/378121773-3df785b0-64b9-4b58-8a1e-d09023914f91.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4ODU0NTAsIm5iZiI6MTcyOTg4NTE1MCwicGF0aCI6Ii83NjU5NTg1NS8zNzgxMjE3NzMtM2RmNzg1YjAtNjRiOS00YjU4LThhMWUtZDA5MDIzOTE0ZjkxLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI1VDE5MzkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzZDFiZTljZWYwMDU5YTM2YTY3ZWM2OWI2ZTU5N2RiYzEyOWI3NjljZjc0ZWEyZWRlMjc5ZDkzZWVmZjFkYzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.o9F-3vpautAtl3SHxSU09uDl7Qu_pEHf9TmDRlzpH9g)

also a thing for catalan

**Comment by fergassss1** [comment]
this setting too
![1000008009](https://github.com/user-attachments/assets/782630ea-9562-42bc-a86e-3be5c5fdd423)

**Comment by fergassss1** [comment]
For Spanish, these options show as untranslated (according to Crowdin)
![Screenshot_2024-11-12-13-44-18-001_com android chrome-edit](https://github.com/user-attachments/assets/c788652f-df79-4065-8fea-5dff52de1b52)
Ill try to translate them

**Comment by fergassss1** [comment]
edit thumbnail: Editar miniatura

Thumbnail not selected: Miniatura no seleccionada

Reset thumbnail: Restablecer miniatura

No thumbnail present: No hay miniatura presente

Removed thumbnail: Miniatura eliminada

Show translation in the second line: Mostrar traducción en la segunda línea

Enter value: Introducir valor
